# Filtered bottom flow in bodyframe.
uint64 timestamp		# time of this estimate, in microseconds since system start

float32 sumx				# Integrated bodyframe x flow in meters
float32 sumy				# Integrated bodyframe y flow in meters

float32 vx 				# Flow bodyframe x speed, m/s
float32 vy				# Flow bodyframe y Speed, m/s
